tests/resources/Harbor-Pages/Project.robot

```diff
@@ -64,14 +64,16 @@ Switch To Replication
     Sleep  1
 
 Back To projects
-    Wait Until Element Is Enabled  xpath=${projects_xpath}
-    Wait Until Element Is Visible  xpath=${projects_xpath}
-    Click Element  xpath=${projects_xpath}
+    ${element}=  Set Variable  xpath=${projects_xpath}  
+    Wait Until Element Is Visible And Enabled  ${element}
+    Click Element  ${element}
     Sleep  2
 
 Project Should Display
     [Arguments]  ${projectname}
-    Page Should Contain Element  xpath=//project//list-project//clr-dg-cell/a[contains(.,'${projectname}')]
+    ${element}=  Set Variable  xpath=//project//list-project//clr-dg-cell/a[contains(.,'${projectname}')]
+    Wait Until Element Is Visible And Enabled  ${element}
+    Page Should Contain Element  ${element}
```

Comment:
Seems the `Page Should Contain Element` isn't needed anymore as if `Wait Until Element Is Visible And Enabled` runs without error the element must be in the page.

Reply:
yes, you're right, I will remove "Page Should Contain Element".